src/lib/ReturnTop/ReturnTop.svelte

```diff
@@ -0,0 +1,46 @@
+<script>
+  import { createEventDispatcher } from "svelte";
+  const dispatch = createEventDispatcher();
+  import "../style/app.css";
+  const arrow = `<svg width="30" height="30" viewBox="0 0 40 40" xmlns="http://www.w3.org/2000/svg">
+<rect width="40" height="40" fill="#fff"/>
+<circle cx="20" cy="20" r="17"/>
+<path d="m20.264 8.4834 13.052 12.918h-26.103l13.051-12.918z" fill="#fff"/>
+<rect x="15.241" y="15.659" width="10.047" height="14.353" fill="#fff"/>
+</svg>`;
+
+  /**
+   * Element ID to scroll to
+   * @type {string}
+   */
+  export let elementId = "";
```

Comment:
I feel like `elementId` either needs to be a required property (ie no default value of empty string), OR there needs to be some functionality to handle what happens if `elementId` isn't set, like scrolling to the top of the page.

Reply:
Let's go with required, thanks for this!